src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -367,19 +368,10 @@ const OrganizationStream = createReactClass({
   },
 
   renderEmpty() {
-    const {environment} = this.state;
-    const message = environment
-      ? tct('Sorry no events match your filters in the [env] environment.', {
```

Comment:
Intentional to lose the environment specific message?

Reply:
Yes, Organization Issue stream has a ton of filter options and there isn't a good reason to favour environments.